packages/shared/src/libs/enums/exception-message.enum.ts

```diff
@@ -8,6 +8,8 @@ const ExceptionMessage = {
 	NO_PERMISSION: "You do not have permission to access this resource.",
 	NO_TOKEN_PROVIDED: "No token provided.",
 	PROJECT_API_KEY_ALREADY_EXISTS: "API key already exists for the project.",
+	PROJECT_API_KEY_NOT_FOUND: "API key not found.",
+	PROJECT_API_KEY_UPDATE_FAILED: "Failed to update project API key.",
```

Comment:
I think the message doesn't align with the format of the other messages, perhaps something like this would confirm to the format better?


```suggestion
	PROJECT_API_KEY_UPDATE_FAILED: "Project API key update failed.",
```